scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -872,16 +867,6 @@ def validate_storage_schema(storage):
                 for i in range(sz):
                     check_type(val[i], str, f"data_devices[{i}]")
 
-def get_cvg_cnt_and_cvg_k8(self):
-    try:
-        cvg = self.storage['cvg']
-        cvg_cnt = len(cvg)
-    except:
-        raise MotrError(errno.EINVAL, "cvg not found\n")
-    # Check if cvg type is list
-    check_type(cvg, list, "cvg")
-    return cvg_cnt, cvg
```

Comment:
Function 'get_cvg_cnt_and_cvg_k8' is not called from anywhere (dead code), so removing it.